### PR TITLE
Deprecate multiple statistics endpoints

### PR DESCRIPTION
**Pull request by ncounter**
Deprecate non-documented statistics endpoints.

### CODE REVIEW

File: src/api/public/apidocs/paths/statistics_deprecated_not_documented.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+get:
+  deprecated: true
+  summary: This endpoint is deprecated
+  description: |
+    This endpoint is deprecated, please do not use it anymore.
```
[the remark this answers is not in the document]
Comment (saraycp) [reply src/api/public/apidocs/paths/statistics_deprecated_not_documented.yaml:5]: I wouldn't give details about the reasons, there can be many of them for different endpoints. But I'm ok if we rephrase it to not be so restrictive. 

I also consider it important to mention the alternative endpoint to the deprecated one, if any.

The sentence could be: 
 "_This endpoint is no longer supported and might not work as expected. As an alternative you can use XYZ endpoint_".
 
 What do you think?

[the remark this answers is not in the document]
Comment (ncounter) [reply src/api/public/apidocs/paths/statistics_deprecated_not_documented.yaml:5]: Mentioning the alternative could be an option, but actually I don't see a good matching with any of the deprecated<->non-deprecated ones.

There are partial matches, I'd rather not highlight to be honest, like the following:

---
- **[deprecated]** `/statistics/added_timestamp/{project_name}/{package_name}`
  - returns the timestamp of a specific project/package when it was added
- **[non deprecated]** `/statistics/latest_added`
  - returns only the latest added ones
---
- **[deprecated]** `/statistics/updated_timestamp/{project_name}/{package_name}`
  - returns the timestamp of a specific project/package when it was updated
- **[non deprecated]** `/statistics/latest_updated`
  - returns only the latest updated ones
---
- **[deprecated]** `/statistics/activity/{project_name}/{package_name}`
  - _activity value is stored for sorting only_
- **[non deprecated]** `/statistics/most_active_packages` and `/statistics/most_active_projects`
  - return a list ordered by the _activity_ value as mentioned above
---
No matching at all instead for:
- `/statistics/active_request_creators/{project_name}`
- `/statistics`
- `/statistics/global_counters`
- `/statistics/maintenance_statistics/{project_name}`
---

So, based on these facts, I'd not mentioning anything at all about reasons why they are deprecated and about possible alternatives. Anything against that?

[the remark this answers is not in the document]
Comment (ncounter) [reply src/api/public/apidocs/paths/statistics_deprecated_not_documented.yaml:5]: Simply _"This endpoint is no longer supported or might not work as expected."_ ?

[the remark this answers is not in the document]
Comment (saraycp) [reply src/api/public/apidocs/paths/statistics_deprecated_not_documented.yaml:5]: perfect!

File: src/api/public/apidocs/OBS-v2.10.50.yaml [hunk 2]
```diff
@@ -443,6 +443,20 @@ paths:
...
+  /statistics/updated_timestamp/{project_name}/{package_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/active_request_creators/{project_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/global_counters:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/maintenance_statistics/{project_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
```
Comment (saraycp) [inline src/api/public/apidocs/OBS-v2.10.50.yaml:459]: Sorry, I didn't realize this before. The endpoints should be displayed alphabetically

**Review by saraycp — CHANGES_REQUESTED** [review]

File: src/api/public/apidocs/OBS-v2.10.50.yaml (hunk 2 above)
Comment (ncounter) [reply src/api/public/apidocs/OBS-v2.10.50.yaml]: You're right, I missed that, good catch. Fixed now

**Review by saraycp — APPROVED** [review]
